src/client/vertxbus.js

```diff
@@ -163,10 +167,17 @@ var vertx = vertx || {};
         }
       } else {
         // Might be a reply message
-        var handler = replyHandlers[address];
-        if (handler) {
+        var handlers = replyHandlers[address];
+        if (handlers) {
           delete replyHandlers[address];
-          handler(body, replyHandler);
+          var handler = handlers.replyHandler;
```

Comment:
@purplefox yes, that was necessary if you want to preserve existing behavior (in case other users are expecting a callback with null for failures)